MAUI/PDF-Viewer/Custom-Bookmark.md

```diff
@@ -66,6 +66,39 @@ Bookmark newBookmark = new Bookmark()
 pdfViewer.CustomBookmarks.Add(newBookmark);
 
 
+{% endhighlight %}
+{% endtabs %}
+
+### Tracking Addition of bookmark:
+
+In the [DocumentLoaded](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.SfPdfViewer.html#Syncfusion_Maui_PdfViewer_SfPdfViewer_DocumentLoadFailed) event handler, subscribe to the `CollectionChanged` event of the [CustomBookmarks](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.SfPdfViewer.html#Syncfusion_Maui_PdfViewer_SfPdfViewer_CustomBookmarks) collection to track changes while addition of custom bookmark in it. Within the `CollectionChanged` event handler, you use the `NotifyCollectionChangedAction` enum to determine if the action is `Add`. If so, you iterate through all items in e.NewItems and log each entry to the debug console using **Debug.WriteLine**.
+
```

Comment:
API linking is required for MSDN APIs also
<img width="472" height="137" alt="image" src="https://github.com/user-attachments/assets/f3569922-b8ae-4798-81e3-2992ad3d5384" />